src/app/styleProvider.js

```diff
@@ -0,0 +1,13 @@
+"use client";
+
```

Comment:
I would name the file in PascalCase, but the naming convention should be defined within the team, then just follow it. Same for the globalStyles.js